code/game/objects/structures/spawner.dm

```diff
@@ -92,3 +92,49 @@
 	name = "wumborian fugu den"
 	desc = "A den housing a nest of wumborian fugus, how do they all even fit in there?"
 	mob_types = list(/mob/living/simple_animal/hostile/asteroid/fugu)
+
+/obj/structure/spawner/nether
+	name = "netherworld link"
+	desc = null //see examine()
+	icon_state = "nether"
+	max_integrity = 50
+	spawn_time = 60 SECONDS
+	max_mobs = 15
+	icon = 'icons/mob/simple/lavaland/nest.dmi'
+	spawn_text = "crawls through"
+	mob_types = list(/mob/living/basic/migo, /mob/living/basic/creature, /mob/living/basic/blankbody)
+	faction = list("nether")
+
+/obj/structure/spawner/nether/Initialize(mapload)
+	. = ..()
+	START_PROCESSING(SSprocessing, src)
+
+/obj/structure/spawner/nether/examine(mob/user)
+	. = ..()
+	if(isskeleton(user) || iszombie(user))
+		. += "A direct link to another dimension full of creatures very happy to see you. [span_nicegreen("You can see your house from here!")]"
+	else
+		. += "A direct link to another dimension full of creatures not very happy to see you. [span_warning("Entering the link would be a very bad idea.")]"
+
+/obj/structure/spawner/nether/attack_hand(mob/user, list/modifiers)
+	. = ..()
+	if(isskeleton(user) || iszombie(user))
+		to_chat(user, span_notice("You don't feel like going home yet..."))
+	else
+		user.visible_message(span_warning("[user] is violently pulled into the link!"), \
+							span_userdanger("Touching the portal, you are quickly pulled through into a world of unimaginable horror!"))
+		contents.Add(user)
+
+/obj/structure/spawner/nether/process(delta_time)
+	for(var/mob/living/living_mob in contents)
+		if(living_mob)
+			playsound(src, 'sound/magic/demon_consume.ogg', 50, TRUE)
+			living_mob.adjustBruteLoss(60 * delta_time)
+			new /obj/effect/gibspawner/generic(get_turf(living_mob), living_mob)
+			if(living_mob.stat == DEAD)
+				var/mob/living/basic/blank
+				blank = new(loc)
```

Comment:
```suggestion
				var/mob/living/basic/blankbody/newmob = new(loc)
```
The code has it such that it will NOT work the way you wanted, it would instead have spawned in the abstract basic mob type. Change the var name in lines under this so that it compiles.